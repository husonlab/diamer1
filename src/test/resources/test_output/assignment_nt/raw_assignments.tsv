6
@read1	2:69 
@read6	
@read2	
@read3	
@read4	1:9 
@read5	
