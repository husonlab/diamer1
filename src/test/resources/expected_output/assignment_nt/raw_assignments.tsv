6
@read1	2:68 
@read6	
@read2	
@read3	
@read4	1:10 
@read5	
